translations/traditionalchinese.json

```diff
@@ -499,6 +499,133 @@
+		"You can only save replays for battles.": "",
+		"This battle can't have hidden replays, because the tournament is set to be forced public.": "",
+		"The replay for this battle is already set to hidden.": "",
+		"${user.name} hid the replay of this battle.": "",
+		"You can only do this in battle rooms.": "",
+		"You can only add a Player to unrated battles.": "",
+		"Player must be set to \"p1\" or \"p2\", not \"${target}\".": "",
+		"User ${name} not found.": "",
+		"User ${name} must be in the battle room already.": "",
+		"This room already has a player in slot ${target}.": "",
+		"${targetUser.name} is already a player in this battle.": "",
+		"${name} was added to the battle as Player ${target.slice(1)} by ${user.name}.": "",
+		"Player 2": "",
+		"Players could not be restored (maybe this battle already has two players?).": "",
+		"This game doesn't support /joingame": "",
+		"This game doesn't support /leavegame": "",
+		"You can only do this in unrated non-tour battles.": "",
+		"User ${this.targetUsername} not found.": "",
+		"${targetUser.name} was kicked from a battle by ${user.name} ${(target ? ": "",
+		"You can only set the timer from inside a battle room.": "",
+		"This game's timer is managed by a different command.": "",
+		"The game timer is OFF.": "",
+		"The game timer is ON (requested by ${[...timer.timerRequesters].join(', ')})": "",
+		"Access denied.": "",
+		"Timer was turned off by staff. Please do not turn it back on until our staff say it's okay.": "",
+		"The timer is already off.": "",
+		"\"${target}\" is not a recognized timer state.": "",
+		"Forcetimer is now OFF: The timer is now opt-in. (set by ${user.name})": "",
+		"Forcetimer is now ON: All battles will be timed. (set by ${user.name})": "",
+		"'${target}' is not a recognized forcetimer setting.": "",
+		"This server requires you to be rank ${groupName} or higher to search for a battle.": "",
+		"Since you have reached ${Config.forceregisterelo} ELO in ${target}, you must register your account to continue playing that format on ladder.": "",
+		"Register": "",
+		"The user '${this.targetUsername}' was not found.": "",
+		"You are locked and cannot challenge unlocked users.": "",
+		"You are banned from battling and cannot challenge users.": "",
+		"You must choose a username before you challenge someone.": "",
+		"This server requires you to be rank ${groupName} or higher to challenge users.": "",
+		"This command does not support specifying multiple users": "",
+		"User \"${this.targetUsername}\" not found.": "",
+		"Provide a valid format.": "",
+		"Please provide a valid format.": "",
+		"The format '${originalFormat.name}' was not found.": "",
+		"Your team is valid for ${format.name}.": "",
+		"Your team was rejected for the following reasons:": "",
+		"Battles are now ${shouldHide ? \"hidden (except to staff)\" : \"visible\"} in your trainer card.": "",
+		"/${cmds.slice(0, i + 1).join(' ')}' is a help command.": "",
```

Comment:
I think we should give this an intermediate variable instead of having this much code in a translatable section.

Reply:
Okay.